pkg/infrastructure/baremetal/variables.go

```diff
@@ -3,15 +3,18 @@ package baremetal
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/openshift/installer/pkg/tfvars"
-	baremetaltfvars "github.com/openshift/installer/pkg/tfvars/baremetal"
 	"os"
 	"path/filepath"
+
+	baremetalhost "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
+	"github.com/openshift/installer/pkg/tfvars"
+	baremetaltfvars "github.com/openshift/installer/pkg/tfvars/baremetal"
 )
 
 const (
 	tfVarsFileName         = "terraform.tfvars.json"
 	tfPlatformVarsFileName = "terraform.platform.auto.tfvars.json"
+	mastersFileName        = "masters.json"
```

Comment:
If this is in the install dir, I think it should be invisible (`.masters.json`).

Reply:
Done